sql/patch-schema-2.sql

```diff
@@ -76,3 +76,55 @@ ALTER TABLE `userfilters` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unico
 ALTER TABLE `users` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 ALTER TABLE `userScores_mv` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 ALTER TABLE `wishlists` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+
+--
+-- Table structure for table `playertimes`
+--
+
+DROP TABLE IF EXISTS `playertimes`;
+CREATE TABLE playertimes (
+  id INT AUTO_INCREMENT,
+  gameid VARCHAR(32) NOT NULL,
+  userid VARCHAR(32) NOT NULL,
+  timevote INT(5) unsigned not null,
```

Comment:
might be good to rename `timevote` to mention the actual time unit used, to avoid conversion errors in the future. `timevote_mins`? `minsvote`?

Reply:
Maybe time_in_minutes? Vote_in_minutes? I did wonder whether we might eventually add any other columns to the table, and what we might need to distinguish it from (e.g. if we store the median of the game as well, in which case "time_in_minutes" might be too generic, but I guess we could change that later. Maybe "vote_in_minutes" is safer.

Reply:
I changed this to vote_in_minutes.